account_loan/model/account_loan_line.py

```diff
@@ -382,3 +390,34 @@ def view_account_invoices(self):
             result['views'] = [(res and res.id or False, 'form')]
             result['res_id'] = self.invoice_ids.id
         return result
+
+    @api.model
+    def _cron_generate_loan_entries(self):
+        date_now = datetime.now().strftime(DEFAULT_SERVER_DATE_FORMAT)
+        lines = self.search([
+            ('has_moves', '=', False),
+            ('has_invoices', '=', False),
+            ('loan_id.state', '=', 'posted'),
+            ('date', '<=', date_now),
+        ])
+        for line in lines:
+            if line.is_leasing:
+                line.generate_invoice()
```

Comment:
You could do `line.filtered(lambda r: r.is_leasing).generate_invoice()` and `line.filtered(lambda r: not r.is_leasing).generate_move()` as functions are available for multiple lines.

Reply:
Thank you @etobella for your comments.
- About version: should we consider this a major update and what the ver. will be? (11.1.0 or 11.0.2).
- About translation: honestly I don't have enough information about OCA and transifex but I like to include the translation of my language for offline use purposes.

Reply:
Yes, I could do that but I think the used code is simpler and there is no difference in result.
Thank you for your suggestion.

Reply:
Well, you are using two fors when you could use one.

Reply:
About translation, I would prefer other people opinions. However, when modules are updated, a good practice is to squash translation commits, but if a translation is made in a regular commit it always gives problems. So, I think you should make the translation in transifex. You can access transifex with these link:
https://www.transifex.com/oca/OCA-account-financial-tools-11-0/

Reply:
@jbeficent @etobella 
I will revert the changes for translation and try to work it with Transifex.